client/src/components/restaurant/restaurant-item.js

```diff
@@ -85,7 +86,7 @@ class Restaurant extends Component {
             <Menu id={id} isAuthenticated={isAuthenticated} />
           </Tab>
           <Tab eventKey="events" title="Events">
-            <h3>Events</h3>
+            <RestaurantEvents id={id} />
```

Comment:
This link doesn't need on this page, could you remove it?
![image](https://user-images.githubusercontent.com/6152066/80856659-63ef7380-8c54-11ea-9d7e-4f7bb79f89a7.png)

Reply:
> This link doesn't need on this page, could you remove it?
> ![image](https://user-images.githubusercontent.com/6152066/80856659-63ef7380-8c54-11ea-9d7e-4f7bb79f89a7.png)

@narekacy no answer to this comment?) link is removed or will stay the same?

Reply:
Sorry, just wanted to remove it and answer that it is done and already pushed. 
Currently I'm working on task #137 as it is important one for this sprint.

I see all comments, sorry for late answers.

Reply:
Done

Reply:
By default, links in Internet have blue color. This style can be misleading users
![image](https://user-images.githubusercontent.com/6152066/80942325-2eca5900-8ded-11ea-9e0b-60641bbcada7.png)

Reply:
Don't remove link from **all events** page. Only from **restaurant events** page (it leads to the same page)
![image](https://user-images.githubusercontent.com/6152066/80942459-781aa880-8ded-11ea-8b94-5409e063ab79.png)

Reply:
I thiink events link shouldn't move user to restaurant page but for event page. According to absense of event page (full card page) there is no neccessary to move user to restaurant page.

Reply:
> By default, links in Internet have blue color. This style can be misleading users

Done

Reply:
> I thiink events link shouldn't move user to restaurant page but for event page. According to absense of event page (full card page) there is no neccessary to move user to restaurant page.

Now we use this functionality, you shouldn't break current functionality
If you want to add event page, you can do special task for it

Reply:
I'm not about adding event page right now)

I say moving user to restaurant page isn't UX that user expects. He awaits to see information about event. So we missed this when this functionality was added and I think nothing is breaked while improved.